client/asset/btc/btc.go

```diff
@@ -534,15 +549,15 @@ func (btc *ExchangeWallet) Connect(ctx context.Context) (*sync.WaitGroup, error)
 		return nil, fmt.Errorf("node software out of date. version %d is less than minimum %d", codeVer, minProtocolVersion)
 	}
 	// Initialize the best block.
-	h, err := btc.node.GetBestBlockHash()
+	h, err := btc.wallet.GetBestBlockHash()
```

Comment:
It's no longer just a wallet. Let's rename the `wallet` field to be more descriptive. Maybe just bring back `node`?